src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy

```diff
@@ -493,7 +493,7 @@ mode with existing flows and hold flows of different table-mode types"() {
     }
 
     @Tags([LOW_PRIORITY])
-    @Ignore("https://github.com/telstra/open-kilda/issues/3961")
+//    @Ignore("https://github.com/telstra/open-kilda/issues/3961")
```

Comment:
same

Reply:
accidentally unignore, test should be ignored, and removed after single table support ends

Reply:
It is still unignored